schemas/mod-data-import-converter-storage/jobProfileSnapshotCollection.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Collection of Profile snapshots",
+  "type": "object",
+  "properties": {
+    "snapshots": {
```

Comment:
To avoid confusion with Snapshot entities better to name the property jobProfileSnapshots